contracts/native/version-control/src/commands.rs

```diff
@@ -60,12 +65,45 @@ pub fn add_modules(
             // Only owner can add modules
             validate_account_owner(deps.as_ref(), &module.provider, &msg_info.sender)?;
         }
-        MODULE_LIBRARY.save(deps.storage, &module, &mod_ref)?;
+        if config.is_testnet {
+            MODULE_LIBRARY.save(deps.storage, &module, &mod_ref)?;
+        } else {
+            PENDING_MODULES.save(deps.storage, &module, &mod_ref)?;
+        }
     }
 
     Ok(VcResponse::action("add_modules"))
 }
 
+/// Approve or decline a module
+pub fn approve_or_decline_module(
+    deps: DepsMut,
+    msg_info: MessageInfo,
+    approves: Vec<ModuleInfo>,
+    rejects: Vec<ModuleInfo>,
+) -> VCResult {
+    // Only Admin can update code-ids
+    ADMIN.assert_admin(deps.as_ref(), &msg_info.sender)?;
+    for module in &approves {
+        let mod_ref = PENDING_MODULES.load(deps.storage, module)?;
+        MODULE_LIBRARY.save(deps.storage, module, &mod_ref)?;
+        PENDING_MODULES.remove(deps.storage, module);
+    }
+    for module in &rejects {
+        PENDING_MODULES.remove(deps.storage, module);
+    }
```

Comment:
Modules included in both approves and rejects will be approved. We should instead reject them first.

Reply:
Additionally, we can skip this check on testnet.

Reply:
on testnet, we skip this step.